src/Surfnet/StepupMiddleware/CommandHandlingBundle/EventListener/CommandProcessingExceptionListener.php

```diff
@@ -18,13 +18,13 @@
 
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\EventListener;
 
-use Surfnet\StepupMiddleware\CommandHandlingBundle\Pipeline\Exception\InvalidCommandException;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Pipeline\Exception\ProcessingAbortedException;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
 
-class InvalidCommandExceptionListener implements EventSubscriberInterface
+class CommandProcessingExceptionListener implements EventSubscriberInterface
```

Comment:
I thought it should handle all Exceptions related to command processing. On the other hand, this listener is specifically for the  `ProcessingAbortedException`, so maybe match that.

Regarding the tech debt issue. Although I'm not sure if I follow what you are saying, I think we're on the same page. I would expect any exception as a result of an API call to be caught at controller level where its output is to be converted to API-specific response data not in some (hidden) listener.